code/game/machinery/doors/airlock.dm

```diff
@@ -989,11 +987,9 @@ About the new airlock wires panel:
 		playsound(src.loc, 'sound/machines/airlockforced.ogg', 30, 1)
 
 	if(autoclose && normalspeed)
-		spawn(150)
-			autoclose()
+		addtimer(src, "autoclose", 150)
 	else if(autoclose && !normalspeed)
-		spawn(11)
-			autoclose()
+		addtimer(src, "autoclose", 11)
```

Comment:
SStimer only has an effective resolution of 5ms. This is irrelevant for most of those, but it might matter here.